js/everything.js

```diff
@@ -532,8 +536,9 @@ $(function() {
 
         // Don't continue appending to the URL if it appears "good enough".
         // This is likely only a problem if the demo link didn't work right the first time
-        if (window.location.href.indexOf(starveTheEgoFeedTheSoulGlitchMob) === -1) {
-            window.location.href = makeListenURL(starveTheEgoFeedTheSoulGlitchMob);
+        if (window.location.href.indexOf(playDemo()) === -1) {
+            window.location.href = makeListenURL(playDemo());
```

Comment:
This is a bug. We should call `playDemo()`, and store it's value before this if statement because we'll get 2 different values back from `playDemo()`.

That aside, I think the if condition may need to change to something like (pseudocode)

```javascript
`if (window.location.href does not contain any element from the demo array) {
    // makeListenURL() ....
}
```